### ISSUE TITLE
Cheatsheet examples

### ISSUE THREAD

**Issue by markknol** [MatthijsKamstra/haxenode#8]
I found this in the cheatsheet: https://github.com/MatthijsKamstra/haxenode/blob/master/haxenode/cheatsheet.md

Type Casting section:
> **JavaScript**
> 
> ```
> var car:Car = vehicle as Car;
> 
> var toString:String = String (10);
> var toNumber:Number = Number ("10");
> var toInteger:int = int (10.1);
> ```

Function types section:

> **JavaScript**
> 
> ```
> function hello (msg:String):void {
> 
> }
> 
> var type:Function = hello;
> ```		

This is not valid JavaScript, Javascript doesn't have types

**Comment by MatthijsKamstra** [comment]
you are completely true... I just copy paste it from http://www.openfl.org/archive/developer/documentation/actionscript-developers/ in this one might have slipped my attention

**Comment by MatthijsKamstra** [comment]
I only just notice you added this to node.js... the same mistake is in js
